src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -325,7 +325,11 @@
 
 helps['acr build-task create'] = """
     type: command
-    short-summary: Creates a new build task with specified parameters.
+    short-summary: Creates a new build definition which can be triggered by git commits or base image updates.
```

Comment:
build definition -> build task?

Reply:
build task is the name, but the description is a build defintion

Reply:
Thanks, just saw the other comment. This look good.